### ISSUE TITLE
最新的develop版有一个地方的源码建议修改

### ISSUE THREAD

**Issue by Stave604671** [PaddlePaddle/Paddle#41687]
### bug描述 Describe the Bug

<img width="652" alt="屏幕截图 2022-04-12 141111" src="https://user-images.githubusercontent.com/59908449/162892506-35db6a69-ed8b-4690-bfcd-450bcdd8f710.png">
我在使用develop版的时候（paddlepaddle==0.0.0），遇上一个bug，bug定位到了这个位置的1719行,检查后发现报错的原因是我的label文件中一共200个标签，label的数值范围是1~200，代码这一块要求label的数值是以0为起点，这一块建议修改一下对这个地方，这个抛出的异常描述看着我有点迷

**Comment by ceci3** [comment]
请问这具体是哪一个loss 函数

**Comment by Stave604671** [comment]
> 请问这具体是哪一个loss 函数

paddle.nn.functional.loss的.py文件里面

**Comment by ceci3** [comment]
嗯嗯，这里面的哪一个function呢？

**Comment by Stave604671** [comment]
> 嗯嗯，这里面的哪一个function呢？
paddle.nn.functional.loss.py文件
cross_entropy函数，源码第1719行，这里没有考虑label数值的起点为1的情况强行报错
